webapp/src/templates/directives/actionbar.html

```diff
@@ -100,7 +100,7 @@
             <div class="dropdown-menu sub-actions" ng-show="actionBar.right.verifyingReport && loadingSubActionBar">
               <div class="loader"></div>
             </div>
-            <div class="dropdown-menu sub-actions" mm-auth="can_verify_reports" ng-show="actionBar.right.verifyingReport && !loadingSubActionBar">
+            <div class="dropdown-menu sub-actions" mm-auth mm-auth-any="['can_verify_reports' && 'can_edit_verification', actionBar.right.selected[0].verified === undefined && 'can_verify_reports']" ng-show="actionBar.right.verifyingReport && !loadingSubActionBar">
```

Comment:
Sorry, I got the syntax wrong the first time:

```
[['can_verify_reports','can_edit_verification'], actionBar.right.selected[0].verified === undefined && 'can_verify_reports']
```

Group permissions in an array, otherwise you're getting the review button when `can_verify_reports` is not allowed but `can_edit_verification` is. 
We're letting angular evaluate the expression and we pass the result to our Auth service: so in this case `'can_verify_reports' && 'can_edit_verification'` -> evaluates to `'can_edit_verification'` so that's the only permission that's effectively required, instead of both. 
Sorry for being incorrect the first time :(

Reply:
Oh interesting.. I forgot to test the case where `can_verify_reports: false`

Reply:
Think this is working now. I've experimented with a few other syntaxes based on the unit tests for `mm-auth-any` and seeing a lot of this error: 

```Uncaught Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!```

Bug or known limitation of this directive?

Reply:
Without knowing the exact syntax, I can't even begin to guess :)